components/Cases/CasesTable.tsx

```diff
@@ -70,21 +70,29 @@ const CaseNoteAction = ({
   personId,
   formType,
   title,
+  deleted,
 }: Case) => (
   <td
     key={`${recordId}-action`}
     className="govuk-table__cell govuk-table__cell--numeric"
     style={{ width: '50px', textAlign: 'center' }}
   >
-    <CaseLink
-      recordId={recordId}
-      externalUrl={caseFormUrl}
-      caseFormData={caseFormData}
-      formName={formName}
-      personId={personId}
-      formType={formType}
-      title={title}
-    />
+    {(formType == 'flexible-form' && !deleted) ||
+    formType != 'flexible-form' ? (
+      <CaseLink
+        recordId={recordId}
+        externalUrl={caseFormUrl}
+        caseFormData={caseFormData}
+        formName={formName}
+        personId={personId}
+        formType={formType}
+        title={title}
+        isImported={false}
+        deleted={false}
+      />
+    ) : (
+      ''
+    )}
```

Comment:
We should show the link to the deleted note if the user is an admin user

Reply:
Now the My Work page should display the deleted case notes entries (together with the link) only if the user is an Admin (or dev)